hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TxnRecordRegression.java

```diff
@@ -101,13 +102,14 @@ private HapiSpec returnsInvalidForUnspecifiedTxnId() {
                 .then(getTxnRecord("").useDefaultTxnId().hasCostAnswerPrecheck(INVALID_ACCOUNT_ID));
     }
 
+    @HapiTest
     private HapiSpec recordNotFoundIfNotInPayerState() {
         return defaultHapiSpec("RecordNotFoundIfNotInPayerState")
                 .given(
                         cryptoCreate("misc").via("success"),
                         usableTxnIdNamed("rightAccountWrongId").payerId("misc"))
                 .when()
-                .then(getTxnRecord("rightAccountWrongId").hasCostAnswerPrecheck(RECORD_NOT_FOUND));
+                .then(getTxnRecord("rightAccountWrongId").hasCostAnswerPrecheck(INVALID_TRANSACTION_ID));
```

Comment:
We should not be changing the response we are returning and comply with mono-service

Reply:
Correct, we need the test to stay the same, and for the modular node behavior to match mono service.

Reply:
Understood. Changes are applied.